tests/py3/test_geo_task.py

```diff
@@ -33,29 +31,22 @@ def test_geo_task(fix_py2env, fix_testDir):
                       z0=0, layer_num=1, thickness=4, litho_base=[substrate])
     wrap2 = Part3DData('Second Layer', 'Sketch007', 'lithography', 'dielectric',
                        layer_num=2, thickness=1)
-    print(os.path.join(fix_testDir, 'py3', 'data', 'geometry_test.fcstd'))
-    # serial_fc_doc = Geometry3D.serialize_fc_file(
+    print(os.path.join(fix_testDir,'py3','data','geometry_test.fcstd'))
     input_file_path = os.path.join(fix_testDir, 'py3', 'data', 'geometry_test.fcstd')
-    freecad_dict = {
-        'pyenv': fix_py2env,
-        'input_file': input_file_path,
-        'params': {'d1': tag1},
-        'input_parts': [block1, block2, sag, wire, shell, block3, substrate, wrap, wrap2]
-    }
-    geo_task = Geometry3D(options=freecad_dict)
 
-    # Run sweeps
-    sweeps = [{tag1: val} for val in np.linspace(2, 7, 3)]
-    result = SweepManager(sweeps).run(geo_task)
+    build_order = [block1, block2, sag, wire, shell, block3, substrate, wrap, wrap2]
+    results = []
+    for d1 in np.linspace(2.,7.,3):
+        built_geo = geometry_3d(fix_py2env,input_file_path,build_order,{'d1':d1})
+        results += [built_geo]
 
     # Investigate results
```

Comment:
Even though this part of the code is unchanged, this would be a good time to clean up the lingering race condition. Looks like a textbook use case for [`tempfile.TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)
```
with tempfile.TemporaryDirectory() as dir_name:
    file_name = os.path.join(dir_name, str(i) + '.fcstd')
    ...
```

Reply:
Done!